### DISCUSSION TITLE
How do you use the new grammar templates in the web ui?

### DISCUSSION THREAD

**Discussion by gregrobison** [oobabooga/text-generation-webui#4109] [Q&A] [answered]
I'm curious about implementing the new grammar feature but can't find how to pick a grammar to use in the web ui. Where are these settings located? Thanks.

**Comment by gregrobison** [comment, marked as answer]
For others who are curious, I found it under Parameters but only when llamacpp is the loader.

**Reply by dwoodev** [reply]
I'd like to know as well.

**Reply by MariusGalyan** [reply]
I searched half a day for an easy implementation of grammar. Thanks! This works :)
